nexus/comment.ts

```diff
@@ -0,0 +1,384 @@
+import { schema } from 'nexus'
```

Comment:
A lot of things in this file were moved from the general `graphql.ts` file. I'll comment to indicate the new stuff